tedana/io.py

```diff
@@ -542,9 +734,8 @@ def load_data(data, n_echos=None):
     img = check_niimg(data)
     (nx, ny), nz = img.shape[:2], img.shape[2] // n_echos
     fdata = utils.load_image(img.get_data().reshape(nx, ny, nz, n_echos, -1, order='F'))
-
     # create reference image
-    ref_img = img.__class__(np.zeros((nx, ny, nz)), affine=img.affine,
+    ref_img = img.__class__(np.zeros((nx, ny, nz, 1)), affine=img.affine,
```

Comment:
It was, because I needed that in order to calculate frequencies appropriately - it gave rise to the other PR.

Reply:
Since it's been merged into the main code base, now, you can grab it by merging in master ! Maybe wait until we're happy with your other changes before doing so, though, just in case we introduce any nasty merge conflicts.

Reply:
Well, didn't run into any merge conflicts so I _think_ I did it right. I'll be doing heaps of testing before I actually start pushing the new commits. Fingers very crossed.

Reply:
Looks good ! :+1: